workspace.md

```diff
@@ -0,0 +1,98 @@
+# Spruce Skills Kit
+
+Lerna workspace for managing skills kit dependencies
+
+```
+├── node_modules/
+├── packages
+│   ├── sprucebot-skills-kit
+│   ├── sprucebot-skills-kit-server
+│   ├── sprucebot-node
+│   └── react-sprucebot
+├── CHANGELOG.md
+├── README.md
+├── lerna.json
+├── package.json
+└── yarn.lock
+```
+
+## Setup
+
+1. `git clone` this repo
+1. `cd workspace.sprucebot-skills-kit`
+1. `git clone` the repos defined in `package.json.workspaces[**]` _TODO - sprucebot-cli could do this for us and setup upstream branches etc_
+1. `yarn install` to install all the dependencies defined in all the cloned workspaces
+1. `cd packages/sprucebot-skills-kit && sprucebot skill register`
+1. `cd ../../`
+1. `yarn local` to start the skills kit using pm2
```

Comment:
Feels like we could move this into a setup script to make initial bootstrapping easier.